### ISSUE TITLE
v 9.3 pre-release - highlight single quote issue

### ISSUE THREAD

**Issue by ziplizard** [Komodo/KomodoEdit#695]
I just downloaded the 9.3 release today and immediately encountered an issue that is very easy to replicate. Within a PHP file, simply highlight any piece of code, then type a single quote (apostrophe) and you will notice the highlighted code is replicated with a single quote after it. The expected behavior would be that the entire highlighted content would be replaced with the single quote.

**Comment by cgchoffman** [comment]
That would be a pretty lucky bug since many people asked for this as a feature ;)

Preferences > Editor > Smart Editing: Disable "Wrap selection with typed deliminter".

Or delete the selection then type the char you wanted to replace the selection with.

**Comment by Naatan** [comment]
Note there are some bugs around this feature at the moment, but to my knowledge they will all be fixed in the next nightly.

**Comment by ziplizard** [comment]
I would think this new feature "Wrap selection with typed delimeter" would be disabled by default - to avoid confusion for users such as myself that are used to the older behavior.

**Comment by cgchoffman** [comment]
I don't agree that disabling this should be default. It's an extremely useful tool.  I know @Naatan won't like this but perhaps a notification that tells you that that command just ran?

**Comment by Naatan** [comment]
I could show a notification for the first time it triggers, like we do for auto-completions.
